iiif_pipeline/pipeline.py

```diff
@@ -92,8 +94,12 @@ def run(self, source_dir, target_dir, skip, replace):
                             file_type, identifier))
                 cleanup_files(identifier, [jp2_dir, pdf_dir, manifest_dir])
             except Exception as e:
-                print(e)
+                print(
+                    "Error processing identifier {} with ref_id: {}".format(
```

Comment:
Is there an incorrect number of variables for this line (and 102)?

Reply:
good catch, thanks!